index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <title>Awesome Books</title>
+</head>
+<body>
+  <header>
+    <div class="top-nav">
+      <span>Awesome Books</span>
+      <nav>
+        <ul>
+          <li><a href="#" id="books-link">List</a></li>
+          <li><a href="#" id="add-book-link">Add Book Form</a></li>
+          <li><a href="#" id="contact-link">Contact</a></li>
+        </ul>
+      </nav>
+
+    </div>
+    <div class="date-container" id="date">
+        
+    </div>
+  </header>
+
+  <main>
+    <section id="books-section" >
+      <h2 class="book-title">All Awesome Books</h2>
+      <div id="books-list" class="book-stack">
+        
+      </div>
+    </section>
+
+    <section id="add-book-section" >
+      <h2 class="book-title">Add Book Form</h2>
+      <form id="add-book-form">
+        <div>
+          <label for="title">Title:</label>
+          <input type="text" id="title" name="title" required>
+        </div>
+        <div>
+          <label for="author">Author:</label>
+          <input type="text" id="author" name="author" required>
+        </div>
+        <div>
+          <label for="genre">Genre:</label>
+          <input type="text" id="genre" name="genre" required>
+        </div>
+        <div>
+          <label for="year">Year:</label>
+          <input type="number" id="year" name="year" required>
+        </div>
+        <button type="submit">Add Book</button>
+      </form>
+    </section>
+
+    <section id="contact-section" >
+      <h2 class="book-title">Contact Information</h2>
+      <p>Do you have any questions or you just want to say "Hello"? <br> You can reach out to us!!</p>
+      <ul>
+        <li>Email: luisemiliorojas@hotmail.com</li>
+        <li>Phone: +57 (318) 775 76 20</li>
+        <li>Address: 123 Street, Neiva, Colombia</li>
+      </ul>
+    </section>
```

Comment:
- [ Required ] I think it would be better if your design for the contact section is similar to the design on the project wireframe for this milestone. Currently, your design varies from the wireframe in the following way.
  
   - [ ] The contact information is not centered in the middle of the page, kindly center it. You can do this using either `text-align: center` or `justify-content: center `

## Your design
![Screenshot from 2023-07-07 02-14-51](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/16392046/28793aa9-3537-4a99-b086-9e9009159088)

## Project wireframe
![awesome_books_full_website](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/16392046/3e0f0a03-7983-4bd5-8b1d-55b8557f2853)